client/orderbook.go

```diff
@@ -6,16 +6,16 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	helpers2 "github.com/1inch/1inch-sdk-go/internal/helpers"
```

Comment:
This import got aliased weirdly

Reply:
helpers will be removed at all, as well as current file